cookbook/install/inestable_versions

```diff
@@ -0,0 +1,95 @@
+How to Install and Use an Inestable Symfony Version
+===================================================
+
+Symfony releases two new minor versions (2.5, 2.6, 2.7, etc.) per year, one in
+May and one in November (:doc:`see releases detail <contributing/community/releases>`).
+Testing the new Symfony versions in your projects as soon as possible is important
+to ensure that they will keep working as expected.
+
+In this article you'll learn how to install and use new Symfony versions before
+they are released as stable versions.
+
+Creating a New Project Based on an Inestable Symfony Version
+------------------------------------------------------------
+
+Suppose that Symfony 2.7 version hasn't been released yet and you want to create
+a new project to test its features. First, :doc:`install Composer </cookbook/composer>`
+package manager. Then, open a command console, enter your projects directory and
+execute the following command:
+
+.. code-block:: bash
+
+    $ composer create-project symfony/framework-standard-edition my_project "2.7.*" --stability=dev
```

Comment:
the problem i am seeing is that people that want to test new features and that their project still works will actually rather do:

```
// change in composer.json and stability=dev in there too
// composer install
```
or
```
rm -rf vendor/symfony/symfony
composer install --prefer-source
cd vendor/symfony/symfony
git checkout VERSION_HERE
```